projects/vdk-core/src/vdk/internal/cli_entry.py

```diff
@@ -158,16 +161,20 @@ def main() -> None:
     """
     # configure basic logging , it's expected that a plugin would override and set it up properly
     click_log.basic_config(logging.getLogger())
+    plugin_load_success = True
 
     log.debug("Setup plugin registry and call vdk_start hooks ...")
     plugin_registry = PluginRegistry()
     plugin_registry.add_hook_specs(InternalHookSpecs)
-    plugin_registry.load_plugins_from_setuptools_entrypoints()
+    try:
+        plugin_registry.load_plugins_from_setuptools_entrypoints()
+    except Exception:
+        plugin_load_success = False
```

Comment:
Why does not `plugin_load_success` exist into `plugin_registry`, seems contextual to me? the way currently, that value is actually lost when going out of method scope...